handlers/idcathandler/idcat.go

```diff
@@ -190,6 +191,18 @@ func (ih *IDcatHandler) Init(opts ...string) error {
 	return nil
 }
 
+// Info returns the handler options and required auth steps.
+func (ih *IDcatHandler) Info() *types.Message {
+	return &types.Message{Title: "idCat", AuthType: "blind", AuthSteps: []*types.AuthField{}}
```

Comment:
nit: I think you can leave out AuthSteps, because in general we should still treat a nil slice just like an empty list.